docs/develop/typescript/failure-detection.mdx

```diff
@@ -1,164 +1,152 @@
-
-```ts
-await client.workflow.start(example, {
-  taskQueue,
-  workflowId,
-  workflowRunTimeout: '1 minute',
-});
-```
-
-<!--SNIPEND-->
-
-<!--SNIPSTART typescript-task-timeout -->
-
-[snippets/src/client.ts](https://github.com/temporalio/samples-typescript/blob/main/snippets/src/client.ts)
-
-```ts
-await client.workflow.start(example, {
-  taskQueue,
-  workflowId,
-  workflowTaskTimeout: '1 minute',
-});
-```
-
-<!--SNIPEND-->
+## Workflow retries {#workflow-retries}
 
-### Workflow retries {#workflow-retries}
+**How to set Workflow retries using the Temporal TypeScript SDK**
 
 A Retry Policy can work in cooperation with the timeouts to provide fine controls to optimize the execution experience.
 
 Use a [Retry Policy](/encyclopedia/retry-policies) to retry a Workflow Execution in the event of a failure.
 
 Workflow Executions do not retry by default, and Retry Policies should be used with Workflow Executions only in certain situations.
 
-Create an instance of the Retry Policy, known as [`retry`](https://typescript.temporal.io/api/interfaces/client.WorkflowOptions/#retry) in TypeScript, from the [`WorkflowOptions`](https://typescript.temporal.io/api/interfaces/client.WorkflowOptions) of the Client interface.
-
-<!--SNIPSTART typescript-retry-workflow -->
-
-[snippets/src/client.ts](https://github.com/temporalio/samples-typescript/blob/main/snippets/src/client.ts)
+The Retry Policy can be set through the [`WorkflowOptions.retry`](https://typescript.temporal.io/api/interfaces/client.WorkflowOptions/#retry) property when starting a Workflow using either the Client or Workflow API.
 
-```ts
+```typescript
 const handle = await client.workflow.start(example, {
   taskQueue,
   workflowId,
   retry: {
     maximumAttempts: 3,
+    maximumInterval: '30 seconds',
   },
 });
 ```
 
-<!--SNIPEND-->
+## Activity Timeouts {#activity-timeouts}
 
-## How to set Activity timeouts {#activity-timeouts}
+**How to set Activity Timeouts using the Temporal TypeScript SDK**
 
-Each Activity timeout controls the maximum duration of a different aspect of an Activity Execution.
+Each Activity Timeout controls the maximum duration of a different aspect of an Activity Execution.
 
-The following timeouts are available in the Activity Options.
+The following Timeouts are available in the Activity Options:
 
-- **[Schedule-To-Close Timeout](/encyclopedia/detecting-activity-failures#schedule-to-close-timeout):** is the maximum amount of time allowed for the overall [Activity Execution](/activities#activity-execution).
-- **[Start-To-Close Timeout](/encyclopedia/detecting-activity-failures#start-to-close-timeout):** is the maximum time allowed for a single [Activity Task Execution](/workers#activity-task-execution).
+- **[Schedule-To-Close Timeout](/encyclopedia/detecting-activity-failures#schedule-to-close-timeout):** is the maximum amount of time allowed for the overall [Activity Execution](/activities#activity-execution);
```

Comment:
I think "entire" is better than "overall" because it relates to the duration and is more specific.  I'd also use the whole from-to (from scheduling to reaching the Closed state or whatever it actually is). It doesn't take many words and it helps differentiate two things that sound really similar when doing a quick visual scan through docs. 

We normally don't use semicolons in lists. We skip punctuation. (I know what you're replacing doesn't skip.)